modyn/supervisor/internal/pipeline_executor/pipeline_executor.py

```diff
@@ -710,19 +710,21 @@ def _evaluate_and_store_results(
         last_timestamp: int,
     ) -> None:
         """Evaluate the trained model and store the results."""
+        # pylint: disable=too-many-locals
         assert self.grpc.evaluator is not None, "Evaluator not initialized."
         assert self.state.pipeline_config.evaluation is not None, "Evaluation config not set."
         s.pipeline_status_queue.put(
             pipeline_stage_msg(PipelineStage.EVALUATE, MsgType.ID, id_submsg(IdType.TRIGGER, trigger_id))
         )
 
-        eval_strategy_config = self.state.pipeline_config.evaluation.eval_strategy
+        eval_strategy_config = self.state.pipeline_config.evaluation.handlers[0].strategy
```

Comment:
I don't get yet why we just take the first handler. Is this PR not using the actual handlers yet?

Reply:
business logic changes were intended in next PR, this should only changes config models.